src/Payum/Core/Request/GetBinaryStatus.php

```diff
@@ -140,6 +142,22 @@ public function isCanceled()
     /**
      * {@inheritDoc}
      */
+    public function markReversed()
```

Comment:
We cannot add those methods here because of BC promise. 
Second I am not sure it has to be added at all. Let me read more about those stuff.